CHANGELOG.md

```diff
@@ -1,34 +1,1209 @@
-# Change Log
```

Comment:
The old changelog was using a history.md for old releases and gcg only for the last few.  I think I can fix this.